Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -4620,7 +4621,11 @@ static bool selectionIsInsideFixedPositionContainer(LocalFrame& frame)
 
             m_dynamicSizeUpdateHistory.clear();
 
-            m_page->setPageScaleFactor(scaleFromUIProcess.value(), scrollPosition, m_isInStableState);
+            if (auto* pluginView = mainFramePlugIn())
+                pluginView->setPageScaleFactor(scaleFromUIProcess.value(), std::nullopt);
```

Comment:
I think the scrollPosition is the origin we're passing to `WebCore::Page::setPageScaleFactor()`, so shouldn't we do likewise when invoking `PluginView::setPageScaleFactor()`?

Reply:
Yes indeed! Changed!